packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCardVerificationTag.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactNode } from 'react';
+import { useAuthentication } from '@deriv/api';
+import VerificationFailedIcon from '../../../../../public/images/ic-verification-failed-status.svg';
+import VerificationPendingIcon from '../../../../../public/images/ic-verification-pending-status.svg';
+import VerificationSuccessIcon from '../../../../../public/images/ic-verification-success-status.svg';
+import { THooks } from '../../../../../types';
+
+type Props = {
+    category: 'poa' | 'poi' | null;
+    icon: ReactNode;
+};
+
+const verificationStatusIconMapper: Partial<
+    Record<
+        NonNullable<THooks.Authentication['poa_status']> | NonNullable<THooks.Authentication['poi_status']>,
+        JSX.Element
+    >
+> = {
+    pending: <VerificationPendingIcon height={8} width={8} />,
+    rejected: <VerificationFailedIcon height={8} width={8} />,
+    verified: <VerificationSuccessIcon height={8} width={8} />,
+};
+
+/**
+ * Component to show the verification status of the user
+ * @param category
+ * @param icon
+ * @returns
+ */
+
+const JurisdictionCardVerificationTag: React.FC<Props> = ({ category, icon }) => {
+    const { data } = useAuthentication();
+
+    const renderStatusIcon = (category: string, status: string) => {
```

Comment:
Lets properly typesafe this. Since THooks.Authentication is repeated multiple times, we can make a general type on top for better readability
```suggestion
    const renderStatusIcon = (
        category: Props['category'],
         status: NonNullable<THooks.Authentication['poa_status' | 'poi_status']>
    ) => {
```